tests/bench/longmem_common.py

```
@@ -0,0 +1,406 @@
+    form "XmYYs" with zero-padded seconds. Otherwise, it returns the
+    duration in seconds with two decimal places, e.g., "12.34s".
+
+    Args:
+        total_seconds: The duration in seconds.
+
+    Returns:
+        A formatted duration string.
+    """
+    minutes = int(total_seconds // 60)
+    if minutes > 0:
+        seconds_rounded = int(round(total_seconds - minutes * 60))
```

Ruff (0.14.10)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)